bbn-test-5/babylon-validators/README.md

```diff
@@ -0,0 +1,304 @@
+
+### Verifying Validator Setup
+
+1. First, get your validator's operator address using your Babylon address: 
+```shell
+babylond keys show <your-key-name> --address --bech val --home <path>--keyring-backend test
+```
+
+For example, for the address we used above is `bbn1qh8444k43spt6m8ernm8phxr332k85teavxmuq`, 
+the operator address is `bbnvaloper1qh8444k43spt6m8ernm8phxr332k85teavxmuq`. 
+
+2. Then, inspect your validator's details: 
+
+```shell 
+babylond query staking validator <validator-operator-address> --home <path>
+```
+
+You should see your validator's configuration, including: 
+
+```yaml 
+validator:
+  commission:
+    rates:
+      current: "100000000000000000" 
+      max: "1000000000000000000" 
+      max_change: "10000000000000000"
+  description:
+    moniker: "my-validator" 
+    website: "https://myweb.site" 
+    security_contact: "my-validator0@gmail.com"
+  status: 1 tokens: "100"
+```
+
+### Understanding Validator Status
+
+Your validator enters the active set based on two conditions: 
+1. The completion of the current epoch (a network-wide time period for 
+coordinating activities) 
+2. Having sufficient stake to qualify for the active set.
+
+When active, your status will show as `BOND_STATUS_BONDED`.
+
+### Managing Your Validator
+
+For delegation operations (`delegate`, `redelegate`, `unbond`, `cancel-unbond`),
+you must use the wrapped messages in the `checkpointing` and `epoching` modules.
+This is because standard staking module messages are disabled in Babylon.
+
+For detailed information about these operations, visit our
+[documentation](https://docs.babylonlabs.io/docs/developer-guides/modules/epoching#delaying-wrapped-messages-to-the-end-of-epochs).
+
+## Advanced Security Architecture
+
+Validators should be run with additional security measures:
+
+**Sentry Node Architecture**  
+For enhanced security, validators should implement a sentry node architecture. 
+This involves deploying sentry nodes as a protective layer around your 
+validator node, following the established [Cosmos Sentry Node Architecture](https://forum.cosmos.network/t/sentry-node-architecture-overview/454). 
+The sentry nodes act as a buffer between your validator and the public network, 
+with a private network maintained between your validator and its sentry nodes. 
+This architecture helps protect your validator from DDoS attacks and other 
+network-level threats by ensuring your validator only communicates with trusted 
+sentry nodes rather than directly with the public network.
+
+## Enhanced Monitoring
+
+In addition to basic node monitoring, validators should:
+
+1. Monitor validator performance metrics
+   - Signing performance
+   - Missed blocks
+   - Voting power changes
+   - BLS signing status
+
+2. Set up alerts for:
+   - Slashing conditions
+   - Validator status changes
+   - Network participation metrics
+   - System resource thresholds
+
+### Prometheus Configuration
```

Comment:
Why do we duplicate all the info here? We can merge it all to the node doc - except for info that's validator-specific.